broken-links-script/Extractlinks.js

```diff
@@ -50,6 +49,13 @@ const shortcodeMapping = {
   "email-c8y-info": "info@cumulocity.com",
 };
 
+const hasRenderFalse = (fileContent) => {
+  const fm = fileContent.match(/(^|\n)---\s*[\s\S]*?\n---/);
+  if (!fm) return false;
+  const block = fm[0];
+  return /_build:\s*[\r\n]+[\s\S]*?render:\s*false\b/i.test(block);
```

Comment:
Here you check the contents of the front matter, right?
```suggestion
  const frontMatter = fm[0];
  return /_build:\s*[\r\n]+[\s\S]*?render:\s*false\b/i.test(frontMatter);
```